packages/esm-messages-app/src/config-schema.ts

```diff
@@ -0,0 +1,13 @@
+import { Type } from '@openmrs/esm-framework';
+
+export const configSchema = {
+  endOfMessageType: {
+    _type: Type.Array,
+    _description: 'Specifies the number of time after which a message service is endend, i.e After 7 times',
```

Comment:
I think this description needs to be improved. I think I understand what this says, (although there are some typos that should be cleaned up, e.g., "endend" -> "ended" and "i.e" -> "i.e.,", but the description makes it sound like it takes a single number, i.e., the number of times to retry sending a message, but the `_type` here is an array and I have no clue what value belongs in there.

It's also not clear to me why this would be specified in the frontend configuration. From the description, these feels like a value that should be set via a global property or something else.